.gitignore

```diff
@@ -1,4 +1,3 @@
-.bundle
```

Comment:
We could keep this but add `!.bundle/config` so that we check in the config but not anything else (not sure what else this directory is used for but it may contain temporary files).